deep_qa/training/text_trainer.py

```diff
@@ -80,6 +80,20 @@ class TextTrainer(Trainer):
         or not.  It's handled correctly for ``num_sentence_words`` and ``num_word_characters`` in
         this class, but you need to be sure to implement it correctly in subclasses for this to
         work.
```

Comment:
Should we perhaps try to avoid parameters which depend on other parameters? `use_dynamic_padding` could be a dict of parameters instead.